server/api/user/user.controller.js

```diff
@@ -340,44 +336,43 @@ exports.authCallback = function(req, res, next) {
 exports.attendance = function(req,res){
     var userId = req.params.id;
     var userCode = req.body.code;
-    console.log("userCode",userCode);
-    console.log("process.env.RCOSDAYCODE",process.env.RCOSDAYCODE);
-    if (! process.env.RCOSDAYCODE){
-
-        res.send(500, "No day code yet!");
-
-    }
-    else if (!userCode || userCode.toUpperCase() != process.env.RCOSDAYCODE.toUpperCase()){
-
-        res.send(500, "Day Code Incorrect!");
-
-    }else{
-
-        var result = User.update({
-            _id: userId
-        },{
-            $push: {
-                attendance: new Date()
-            }
-        }, function(err){
-            if (err) return res.send(500, err);
-            res.send({"success":(err !== 0)});
-
-        });
-    }
+   
```

Comment:
We currently don't have any time stamp on the daycode. But it only adds the date that it was entered so if we haven't replaced the day code yet and they submit on a different day it will show the day submitted.